cypress/integration/resources.spec.js

```diff
@@ -227,6 +231,16 @@ describe('resources', () => {
     checkAllCardsForDataAttribute(DATA_TEST_LANGUAGES, 'JavaScript');
   });
 
+  it('opens and closes login modal', () => {
+    cy.findByTestId(BUTTON).click();
+    cy.findByTestId(MODAL_CONTENT).should('exist').and('be.visible');
+    cy.findAllByTestId(INPUT).should('have.length', 3);
```

Comment:
```suggestion
```

I think you're doing enough relevant coverage with just L238. L238 is also less likely to break in the future if we tweak the login modal's content.